lib/eco/ip_plugins/pythonindentation.py

```diff
@@ -13,12 +13,68 @@ def incparse_init(self):
         self.comment_tokens = []
         self.indent_stack = None
 
+    def right_sibling(self, node):
+        """Checks if a new indentation token was added or removed during the
+        current parse and returns the correct next node according to the current
+        parse tree instead of the previous version of the tree."""
+        if node in self.lookahead:
+            return self.lookahead[node]
+        else:
+            n = node.right_sibling(self.incparser.prev_version)
+            if n in self.deleted:
+                return self.right_sibling(n)
+            return n
+
+    def wasdeleted(self, node):
```

Comment:
I should be. But the ip_plugins are dead anyways, so there's no need to fix this.